Floatops undefined references

Where should log,exp,pow,tanh, etc. come from? I get many errors like this when trying to build the ESP32 version while building floatops.o in function floatop:

```
esp32/cforth/build/esp32/../../src/cforth/floatops.c:101: undefined reference to `log' 
```

--- a/build/esp32/Makefile
+++ b/build/esp32/Makefile
@@ -6,6 +6,7 @@ TOPDIR=../..
 
 CONFIG += -DBITS32
 CONFIG += -DFLOATING -DMORE_FP
+LIBS += -lm
 
 CFLAGS += -m32
 

